### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by sudach**
## What?
- Script to install in an independent environment.
```bash
$ python install_pydtk.py
```

## Why?
- To install tools such as cli without changing the existing environment.

## See also [Optional]
- https://github.com/dataware-tools/pydtk/issues/82

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/87#" title="Author self-approved">sudach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [sudach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by d-hayashi — APPROVED** [review]
Great!
Thank you for your contribution!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/87#pullrequestreview-725980744" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/87#" title="Author self-approved">sudach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,sudach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
